facia/app/conf/HealthCheck.scala

```diff
@@ -8,3 +8,13 @@ object HealthCheck extends AllGoodHealthcheckController(9008, "/uk") {
     NoCache(if (isOk) Ok("OK") else ServiceUnavailable("Service Unavailable"))
   }
 }
+
+object HealthCheckController extends CachedHealthCheckController {
+  override val paths = Seq("/uk/business")
```

Comment:
Instead of checking `/uk` which is quite heavy, I changed it to `/uk/business` let me know what you think